test/database.js

```diff
@@ -296,4 +296,9 @@ describe('database', function() {
     await db.close();
     await db2.close();
   });
+
+  it('should execute admin commands', async() => {
+    const currentOps = await db.adminCommand({ currentOp: 1 });
```

Comment:
any idea why this is failing for a bunch of environments?

Reply:
It's because I used a command not present in MongoDB 2.6 in tests (`currentOp`). Looking for a suitable replacement.

Reply:
Might we decide to just drop support for earlier versions of MongoDB?

Reply:
We might soon, but we're getting it for free at the moment; I initially thought this was a failure due to non-test code and was about to propose dropping support for older versions as well.